### PR TITLE
prepare version 2.0.0 release

### PR DESCRIPTION
**Pull request by ElectricMaxxx**
When reading throug our docmentation book, i realized that we should also release this one here as `syfmony-cmf/symfony-cmf` is still released.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Makefile symfony-cmf/standard-edition
+
+test:
```
Comment (ElectricMaxxx) [inline Makefile]: first step into the Symfony 4.0 future, but sonata uses make files too.

File: composer.json [hunk 2]
```diff
@@ -43,9 +43,8 @@
         "liip/functional-test-bundle": "~1.0",
         "sensio/generator-bundle": "~3.0",
         "incenteev/composer-parameter-handler": "~2.0",
-        "symfony/phpunit-bridge": "~3.0"
+        "symfony/phpunit-bridge": "^3.3"
```
Comment (ElectricMaxxx) [inline composer.json]: oh it was still there ..., but does not matter right?

File: .travis.yml [hunk 3]
```diff
@@ -3,7 +3,6 @@ language: php
 php:
   - 5.6
   - 7.0
-  - hhvm
```
Comment (ElectricMaxxx) [inline .travis.yml]: Travis does not support hhvm on native containers anymore.

Comment (dbu) [inline .travis.yml]: in the other projects, i added this to the `matrix` section:

```
- php: hhvm
  dist: trusty
```

that worked fine so far.

File: .travis.yml [hunk 4]
```diff
@@ -22,7 +21,7 @@ matrix:
       env: COMPOSER_CMD=install
 
 before_install:
-  - if [[ "$TRAVIS_PHP_VERSION" != "hhvm" ]]; then echo "memory_limit = -1" >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini; fi
+  - echo "memory_limit = -1" >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
```
Comment (dbu) [inline .travis.yml]: please revert when you add hhvm to the matrix with the trusty dist.

File: composer.json (hunk 2 above)
Comment (dbu) [inline composer.json]: s/still/already? but yeah, whatever works for this, its only a dev requirement so if it builds fine on all supported platforms, lets use this version.

**Comment by dbu** [comment]
thanks a lot!

**Pull request merged**